src/app/users/users-update/users-update.component.html

```diff
@@ -1,62 +1,70 @@
+        <mat-form-field>
+          <input matInput i18n-placeholder placeholder="Middle Name" formControlName="middleName">
+        </mat-form-field>
+        <mat-form-field>
+          <input matInput i18n-placeholder placeholder="Last Name" formControlName="lastName" required>
+          <mat-error><planet-form-error-messages [control]="editForm.controls.lastName"></planet-form-error-messages></mat-error>
+        </mat-form-field>
+      </div>
+      <div>
+        <mat-form-field>
+          <input matInput i18n-placeholder placeholder="Login" formControlName="name">
+        </mat-form-field>
+        <mat-form-field>
+          <input matInput i18n-placeholder placeholder="Email" formControlName="email" required>
+          <mat-error><planet-form-error-messages [control]="editForm.controls.email"></planet-form-error-messages></mat-error>
+        </mat-form-field>
+        <mat-form-field>
+          <input matInput i18n-placeholder placeholder="Language" formControlName="language" required>
+          <mat-error><planet-form-error-messages [control]="editForm.controls.language"></planet-form-error-messages></mat-error>
+        </mat-form-field>
+      </div>
+      <div>
+        <mat-form-field>
+          <input matInput i18n-placeholder placeholder="Phone Number" formControlName="phoneNumber" required>
+          <mat-error><planet-form-error-messages [control]="editForm.controls.phoneNumber"></planet-form-error-messages></mat-error>
+        </mat-form-field>
+        <mat-form-field>
+          <input matInput type="date" i18n-placeholder placeholder="BirthDate" formControlName="birthDate" required>
+          <mat-error><planet-form-error-messages [control]="editForm.controls.birthDate"></planet-form-error-messages></mat-error>
+        </mat-form-field>
+        <mat-form-field>
+          <mat-select i18n-placeholder placeholder="Gender" formControlName="gender" required>
+            <mat-option value="male">Male</mat-option>
+            <mat-option value="female">Female</mat-option>
+          </mat-select>
+          <mat-error><planet-form-error-messages [control]="editForm.controls.gender"></planet-form-error-messages></mat-error>
+        </mat-form-field>
+      </div>
+      <div>
+        <mat-form-field>
+          <mat-select i18n-placeholder placeholder="level" formControlName="level" required>
+            <mat-option *ngFor="let level of educationLevel" [value]="level">
+              {{level}}
+            </mat-option>
+          </mat-select>
+          <mat-error><planet-form-error-messages [control]="editForm.controls.level"></planet-form-error-messages></mat-error>
+        </mat-form-field>
+        <label>File Upload:</label>
+        <input type="file" (change)="bindFile($event)" />
+      </div>
+      <div>
+        <button type="submit" mat-raised-button color="primary" i18n>Update</button>
+        <button type="button" mat-raised-button color="warn" (click)="cancel()" i18n>Cancel</button>
```

Comment:
cancel and goback on topbar should do the same thing so we can use same function